cf_remote/commands.py

```diff
@@ -484,6 +485,25 @@ def init_cloud_config():
     print("Config file %s created, please complete the configuration in it." % CLOUD_CONFIG_FPATH)
     return 0
 
+def show_spawned():
+    if not os.path.exists(CLOUD_STATE_FPATH):
+        print("No saved cloud state info")
+        return 1
+
+    vms_info = read_json(CLOUD_STATE_FPATH)
+    vms = []
+    for group_name in vms_info:
+        vms += [{'platform': group_name}]
```

Comment:
The group name doesn't generally have to be the platform, it can be an arbitrary name.

Reply:
I've changed / fixed this.